v2/emailpassword/custom-ui/init/frontend.mdx

```diff
@@ -38,13 +39,17 @@ npm i -s supertokens-web-js
 
 You need to add all of the following scripts to your app
 
+<WebJsInjector>
+
 ```bash
-<script src="https://cdn.jsdelivr.net/gh/supertokens/supertokens-web-js/bundle/website.js"></script>
-<script src="https://cdn.jsdelivr.net/gh/supertokens/supertokens-web-js/bundle/supertokens.js"></script>
-<script src="https://cdn.jsdelivr.net/gh/supertokens/supertokens-web-js/bundle/session.js"></script>
-<script src="https://cdn.jsdelivr.net/gh/supertokens/supertokens-web-js/bundle/^{codeImportRecipeName}.js"></script>
+<script src="^{jsdeliver_webjs_website}"></script>
```

Comment:
i dont think we need this script actually. Since importing session should pull in supertokens-website repo as part of that bundle.